pkg/updatestrategy/drain_test.go

```diff
@@ -24,7 +24,10 @@ const (
 
 func removePod(client kubernetes.Interface, pod v1.Pod) error {
 	var zero int64
-	client.CoreV1().Pods(pod.GetNamespace()).Delete(pod.GetName(), &metav1.DeleteOptions{GracePeriodSeconds: &zero})
+	err := client.CoreV1().Pods(pod.GetNamespace()).Delete(pod.GetName(), &metav1.DeleteOptions{GracePeriodSeconds: &zero})
```

Comment:
If the pod doesn't exist then this returns an `err` here. Looks like the original code would like to just ignore it.